### DISCUSSION TITLE
blivet/libbytesize - fedora weblate - Terms and units have been standardized.

### DISCUSSION THREAD

**Discussion by simmon-nplob** [storaged-project/blivet#1189] [General]
blivet/libbytesize

blivet-libbytesize, which is used to specify capacity during the installation process, has been changed to a single unit and a single term to prevent confusion even if the languages are different.

**Comment by vojtechtrefny** [comment]
I don't understand. The point of localization is localization, with this change you basically removed localization from libbytesize, why? Do you have an example (bug report for example) where translated units are a problem? The units were translated (or translatable) in Anaconda even before libbytesize and I don't remember any issue with this.

**Reply by simmon-nplob** [reply]
In terms of localization of translation, that's true. In this situation, even if there are no errors, I would like to exclude the libbytesize unit system and size from the translation - "**prevent confusion even if the languages are different.**"

**Reply by vojtechtrefny** [reply]
Sorry, but this doesn't explain anything. What confusion are you trying to prevent? I am asking again: **Do you have an example (bug report for example) where translated units are a problem?** Libbytesize is more than 8 years old project and it included localization support from the begging and even before that units were translated in Blivet itself (before splitting the size handling into a separate project) and I am not aware of any issues or bugreports where it would be a problem.

"prevent confusion even if the languages are different" is not a valid argument for me, I could use the same argument to argue *for* localization and not *against* it.

**Reply by simmon-nplob** [reply]
I proposed my opinion that fits the discussion.

In the rawhide test section and the btree-file system that changes in fedora bluesilver, I am checking for inappropriate status in the terminology changes I applied. **If you want to maintain it, you just have to push on.** 

In the translation, sometimes it is necessary to fix it so that it does not change.
I believe my opinion will be reviewed again by translators and participants for each language.

**thanks.**.by simmon

**Comment by simmon-nplob** [comment]
L10N Korean Team maintains the settings proposed through official translations.
Please apply the changed translation.
